gce/load-test.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+set -o nounset
+set -o pipefail
+set -o xtrace
+
+# When running in prow, the working directory is the root of the test-infra
+# repository.
+
+# Taint the Linux nodes to prevent the test workloads from landing on them.
+# TODO: remove this once the issue is resolved:
+# https://github.com/kubernetes/kubernetes/issues/69892
+LINUX_NODES=$(kubectl get nodes -l beta.kubernetes.io/os=linux -o name)
+LINUX_NODE_COUNT=$(echo ${LINUX_NODES} | wc -w)
+for node in $LINUX_NODES; do
+  kubectl taint node $node node-under-test=false:NoSchedule
+done
+
+# Untaint the windows nodes to allow test workloads without tolerations to be
+# scheduled onto them.
```

Comment:
Would it make more sense to just run the load test pods/workloads with a toleration for node.kubernetes.io/os:NoSchedule applied? I don't recall how you did this in your perf-tests PR.

Not a big deal either way.

Reply:
In [perf-test PR](https://github.com/kubernetes/perf-tests/pull/930), using `windows_override.yaml` to override the variables for windows nodes. Thought about using nodeselector + toleration, but not sure if we can override the [tolerations](https://github.com/YangLu1031/perf-tests/blob/60c29768e3d85dbc27426f2414bfda8ef80cb938/clusterloader2/testing/node-throughput/rc.yaml#L26) without affecting the linux load test